src/notation/internal/notationinteraction.cpp

```diff
@@ -26,6 +26,7 @@
 #include <QRectF>
 #include <QPainter>
 #include <QClipboard>
+#include <QApplication>
```

Comment:
Without this you can't compile just this file on its own and you get errors highlighted in Visual Studio.  Not quite sure why it compiles as part of the full build!